waku/waku.md

```diff
@@ -223,9 +221,15 @@ The status message MUST contain an association list containing various options.
 
 This packet is used for sending the standard Waku envelopes.
 
-#### PoW Requirement
+#### Status Update
 
-This packet is used by Waku nodes for dynamic adjustment of their individual PoW requirements. Recipient of this message should no longer deliver the sender messages with PoW lower than specified in this message.
+The Status Update message is used to communicate an update of the settings of the node.
+The format is the same as the Status message, all fields are optional.
+Fields that are omitted are considered unchanged.
+
```

Comment:
Should we still add here something that it is advised not to add fields that don't require an update?